imports/plugins/core/product/server/no-meteor/schemas/schema.graphql

```diff
@@ -6,3 +6,68 @@ type ProductConfiguration {
   "The ProductVariant ID"
   productVariantId: ID!
 }
+
+"Operation"
+type Operation {
+  "Id of document"
+  _id: String
```

Comment:
Would we ever need to add more that an ID? Can we skip having the type and just have a `documentId` property on `WriteError`?

Reply:
removed.